### ISSUE TITLE
ERROR TypeError: "this.passwordStrength is undefined" when disabling registration tab

### ISSUE THREAD

**Issue by kuncevic** [AnthonyNahas/ngx-auth-firebaseui#286]
# Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions
```
Angular CLI: 8.1.1
Node: 10.14.2
OS: darwin x64
Angular: 8.1.1
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router, service-worker

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.801.1
@angular-devkit/build-angular     0.801.1
@angular-devkit/build-optimizer   0.801.1
@angular-devkit/build-webpack     0.801.1
@angular-devkit/core              8.1.1
@angular-devkit/schematics        8.1.1
@angular/cdk                      8.0.2
@angular/fire                     5.2.1
@angular/flex-layout              8.0.0-beta.26
@angular/material                 8.0.2
@angular/pwa                      0.801.1
@ngtools/webpack                  8.1.1
@nguniversal/express-engine       7.1.1
@schematics/angular               8.1.1
@schematics/update                0.801.1
rxjs                              6.5.2
typescript                        3.4.5
webpack                           4.35.2

```

### Repro steps
Disable registration tab as per https://github.com/AnthonyNahas/ngx-auth-firebaseui/issues/202 by [registrationEnabled]=false

`npm start`

see the error in console log

### The log given by the failure
```
ERROR TypeError: "this.passwordStrength is undefined"
    ngAfterViewInit ngx-auth-firebaseui.js:593
    Angular 31
    RxJS 5
    Angular 8
        emit
        checkStable
        onHasTask
        hasTask
        _updateTaskCount
        _updateTaskCount
        runTask
        drainMicroTaskQueue
```

### Desired functionality
Disabling registration tab should not give any errors.

**Comment by AnthonyNahas** [comment]
@kuncevic thank you 👍 

I've just fixed that and patched the lib to v3.2.2 and is now [live](https://www.npmjs.com/package/ngx-auth-firebaseui)

cheers and have a nice sunday 🍻

**Comment by kuncevic** [comment]
@AnthonyNahas works great, thanks for taking such a quick action to solve that!
